templates/pro.html

```diff
@@ -0,0 +1,540 @@
+                  {{ image(url="https://assets.ubuntu.com/v1/05c01589-redis logo.png",
+                            alt="",
+                            width="252",
+                            height="253",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+                  }}
+                </div>
+                <div class="p-logo-section__item u-sv1">
+                  {{ image(url="https://assets.ubuntu.com/v1/850af73c-ruby-rails logo.png",
+                            alt="",
+                            width="240",
+                            height="253",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+                  }}
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col">
+          <div class="p-section--shallow">
+            <h2 class="p-text--small-caps">コンプライアンス：</h2>
+          </div>
+          <div class="row">
+            <div class="col-3">
+              <ul class="p-list--divided u-no-margin--bottom">
+                <li class="p-list__item u-no-padding--bottom">
+                  Defense Information
+                  <br />
+                  Systems Agency (DISA-STIG)
+                </li>
+                <li class="p-list__item u-no-padding--bottom">FedRamp</li>
+                <li class="p-list__item u-no-padding--bottom">NIST</li>
+                <li class="p-list__item u-no-padding--bottom">Sarbanes-Oxley (SOX)</li>
+                <li class="p-list__item u-no-padding--bottom">HIPAA</li>
+                <li class="p-list__item u-no-padding--bottom">FISMA</li>
+              </ul>
+            </div>
+            <div class="col-3">
+              <ul class="p-list--divided">
+                <li class="p-list__item u-no-padding--bottom">Center for Internet Security (CIS)</li>
+                <li class="p-list__item u-no-padding--bottom">Common Criteria</li>
+                <li class="p-list__item u-no-padding--bottom">PCI</li>
+                <li class="p-list__item u-no-padding--bottom">ISO 27001</li>
+                <li class="p-list__item u-no-padding--bottom">FIPS 140</li>
+                <li class="p-list__item u-no-padding--bottom">Cyber Essentials</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="p-section">
+      <div class="row--50-50">
+        <hr class="u-no-margin--bottom is-muted" />
+        <div class="col">
+          <h2>チームに安心感のある運用を</h2>
+        </div>
+        <div class="col">
+          <p>
+            業務に対応する安定性、セキュリティ、コンプライアンス。CVE露出期間を平均98日から1日に短縮。
+          </p>
+          <div class="p-section--shallow">
+            <div class="row">
+              <div class="col-2">
```

Comment:
Making these nested ones col-3s instead of col-2s and 4s respectively aligns the strong text much better (although this isn't on you, this alignment looks way off even in the design  
![image](https://github.com/user-attachments/assets/5f849947-65bb-4e65-a0f9-3c3cbc8be953))

before with col-2s and 4s:
![image](https://github.com/user-attachments/assets/8acdd0d7-96b3-4153-b9fa-37150debb385)

after:
![image](https://github.com/user-attachments/assets/7b42ec7f-a99c-4d9a-9c01-b0d8826dc656)